lua/luasnip/extras/fmt.lua

```diff
@@ -44,6 +45,7 @@ local function interpolate(fmt, args, opts)
 	local defaults = {
 		delimiters = "{}",
 		strict = true,
+		dup_copy = true,
```

Comment:
I feel like `repeat_duplicates` is a bit more descriptive, wdyt?

Reply:
yes, I thought `dup_copy` is good enough but `repeate_duplicates` works also